python/xorbits/_mars/services/storage/api/oscar.py

```diff
@@ -93,10 +93,14 @@ async def is_seekable(self, storage_level: StorageLevel = None) -> bool:
 
     @mo.extensible
     async def get(
-        self, data_key: str, conditions: List = None, error: str = "raise"
+        self,
+        data_key: str,
+        conditions: List = None,
+        cpu: Optional[bool] = None,
```

Comment:
`cpu` is confusing for me, but I don't think of a better name.

Reply:
Maybe `to_cpu`? but `cpu` is OK to me.

Reply:
The meaning of this parameter is moving data in vram to ram, so how about `to_main_mem`?

Reply:
`to_cpu` is OK.